config.rdf

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
+<rdf:RDF 
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+
+    <owl:Ontology rdf:about="http://fedora.info/definitions/v4/config#">
+        <rdfs:label xml:lang="en">Fedora Commons Configuration Ontology</rdfs:label>
+        <rdfs:comment xml:lang="en">Ontology for user-defined configuration properties and classes on Fedora-curated resources.</rdfs:comment>
+    </owl:Ontology>
+
+    <owl:DatatypeProperty rdf:about="http://fedora.info/definitions/v4/config#hasIndexingTransformation">
+        <rdfs:label xml:lang="en">has indexing transformation</rdfs:label>
+        <rdfs:domain rdf:resource="http://fcrepo.info/definitions/v4/repository#Container"/>
```

Comment:
I think that's a fine idea. (Although I'm honestly not too clear on the
differences between fcrepo.org, fedora.info, etc.) And if we were to do
that, it would make a lot of sense for the "source" for that published
ontology to be managed here, in its own repo, and paralleled by some
documentation on the wiki explaining what an indexing component should do
to respect the use of that ontology.

On Tue, Jan 20, 2015 at 4:10 PM, Aaron Coburn <notifications@github.com>
wrote:

> In config.rdf
> <https://github.com/fcrepo4/ontology/pull/13#discussion-diff-23256067>:
>
> > +<?xml version="1.0"?>
> > +<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
> > +<rdf:RDF
> > +     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
> > +     xmlns:owl="http://www.w3.org/2002/07/owl#"
> > +     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
> > +     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
> > +
> > +    <owl:Ontology rdf:about="http://fedora.info/definitions/v4/config#">
> > +        <rdfs:label xml:lang="en">Fedora Commons Configuration Ontology</rdfs:label>
> > +        <rdfs:comment xml:lang="en">Ontology for user-defined configuration properties and classes on Fedora-curated resources.</rdfs:comment>
> > +    </owl:Ontology>
> > +
> > +    <owl:DatatypeProperty rdf:about="http://fedora.info/definitions/v4/config#hasIndexingTransformation">
> > +        <rdfs:label xml:lang="en">has indexing transformation</rdfs:label>
> > +        <rdfs:domain rdf:resource="http://fcrepo.info/definitions/v4/repository#Container"/>
>
> @ajs6f <https://github.com/ajs6f> what would you think of a proposal to
> publish an indexing# ontology in the fedora.info domain?
>
> Or do you see that as being the responsibility of repository
> implementations?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fcrepo4/ontology/pull/13/files#r23256067>.
>